### PR TITLE
Directory Refresh & Modal Autofill

### LINKED ISSUE #91: Autofill Resources with Edit Resource Modal
**Is your feature request related to a problem? Please describe.**

Currently, the modal for edit resource shows empty fields: 
image

It would be nice if they could be autofilled so the LAH staff can quickly make one-off edits (i.e. change a letter here and there).
This will likely involve passing the resource info as props to the card, and then this can be managed using a [controlled component](https://reactjs.org/docs/forms.html#controlled-components).

### PR DESCRIPTION
**Pull request by laurenho025**
## Description
Directory now force refreshes after submitting edit/add form. Modal also has pre-populated inputs when editing a resource.

Fixes #91 

## Screenshots
Screen Shot 2019-12-04 at 3 12 24 PM

### CODE REVIEW

**Comment by alan-fang** [comment]
> You are pushing commits at a very fast pace (across the whole organization).
> Due to that, we cannot deploy the commit [35dfd07](https://github.com/hack4impact-uiuc/life-after-hate/commit/35dfd07bd952ef484c946f1e454628ad88b9026a).
> 
> You can try again later or [upgrade](https://zeit.co/teams/hack4impact/settings/plan) your plan.

lmao

@josh-byster is there a way to manually deploy later?

File: frontend/src/pages/MapView/index.jsx [hunk 1]
```diff
@@ -52,6 +52,8 @@ class MapView extends Component {
   searchHandler = async () => {
     let searchResults;
     try {
+      // search only passes in keyword
+      // Pass in location too
```
Comment (alan-fang) [inline frontend/src/pages/MapView/index.jsx]: let's put any work on this in its own PR

Comment (laurenho025) [reply frontend/src/pages/MapView/index.jsx]: yessir
